quantity.hpp

```diff
@@ -292,36 +531,36 @@ class quantity {
 	 */
 	template<typename U, typename S, typename V, typename T>
 	constexpr quantity& operator+=(const quantity<U,S,V,T>& rhs)          {
-		                                                                      static_assert( are_units_compatible<unit_type , U>::value, "fundamentally incompatible units"      );
-		                                                                      static_assert( std::is_same        <value_type, V>::value, "different value types (use value_cast)");
-		                                                                      static_assert( std::is_same        <tag_type  , T>::value, "incompatible tags (use tag_cast)"      );
+		                                                                      static_assert(are_units_compatible_v<unit_type , U>, "fundamentally incompatible units"      );
+		                                                                      static_assert(detail::is_same_v     <value_type, V>, "different value types (use value_cast)");
+		                                                                      static_assert(are_tags_compatible_v <tag_type  , T>, "incompatible tags (use tag_cast)"      );
 		                                                                      value += rhs.template get_scaled<scale_type>();
 		                                                                      return *this;
 	                                                                      }
 	template<typename U, typename S, typename V, typename T>
 	constexpr quantity& operator-=(const quantity<U,S,V,T>& rhs)          {
-		                                                                      static_assert( are_units_compatible<unit_type , U>::value, "fundamentally incompatible units"      );
-		                                                                      static_assert( std::is_same        <value_type, V>::value, "different value types (use value_cast)");
-		                                                                      static_assert( std::is_same        <tag_type  , T>::value, "incompatible tags (use tag_cast)"      );
+		                                                                      static_assert(are_units_compatible_v<unit_type , U>, "fundamentally incompatible units"      );
+		                                                                      static_assert(detail::is_same_v     <value_type, V>, "different value types (use value_cast)");
+		                                                                      static_assert(are_tags_compatible_v <tag_type  , T>, "incompatible tags (use tag_cast)"      );
```

Comment:
same here, substracting quantities requires _exactly_ the same tag including exponent.

Reply:
Agreed. Done.